src/model/propsContextModel.tsx

```diff
@@ -0,0 +1,3 @@
+export interface propsModel{
```

Comment:
Capital letter. What is this for anyway?

Reply:
Without this interface typescript return bug.